packages/cli/src/cmds/unpublish.ts

```diff
@@ -33,7 +33,7 @@ program
 
 /**
  * Retrieves and validates the ReChunk configuration, ensuring that the specified chunk
- * is present as an entry point in `rechunk.json`.
+ * is present as an entry point in `.rechunkrc.json`.
  *
```

Comment:
🟡 just remind that we should remove the `rc.entry` from the `getValidatedConfig`